### PR TITLE
SBP-2849 | Modified the TwitterActivityUtil to ensure that when the l…

### PR DESCRIPTION
**Pull request by robdouglas**
…ocation extension is populated, it builds off of any existing location map (to prevent the dropping of previously set location data)

### CODE REVIEW

File: streams-contrib/streams-provider-twitter/src/test/java/org/apache/streams/twitter/processor/FetchAndReplaceTwitterProcessorTest.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        FetchAndReplaceTwitterProcessor fetchAndReplaceTwitterProcessor = new FetchAndReplaceTwitterProcessor();
+        fetchAndReplaceTwitterProcessor.prepare(null);
+
+        activity = new Activity();
+        activity.setAdditionalProperty("extensions", extensions);
+
+        fetchAndReplaceTwitterProcessor.replace(activity, json);
+
+        String country = getCountry(activity);
+        Assert.assertTrue(country.equals("us"));
```
Comment (sieverssj) [inline streams-contrib/streams-provider-twitter/src/test/java/org/apache/streams/twitter/processor/FetchAndReplaceTwitterProcessorTest.java]: Please use assertEquals() or assertThat() to test equality.  That way if it fails, the message will spit out the values that weren't equal.

**Comment by robdouglas** [comment]
Responded to code review feedback

File: streams-contrib/streams-provider-twitter/src/test/java/org/apache/streams/twitter/processor/FetchAndReplaceTwitterProcessorTest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        FetchAndReplaceTwitterProcessor fetchAndReplaceTwitterProcessor = new FetchAndReplaceTwitterProcessor();
+        fetchAndReplaceTwitterProcessor.prepare(null);
+
+        activity = new Activity();
+        activity.setAdditionalProperty("extensions", extensions);
+
+        fetchAndReplaceTwitterProcessor.replace(activity, json);
+
+        String country = getCountry(activity);
+        Assert.assertEquals(country, "us");
```
Comment (sieverssj) [inline streams-contrib/streams-provider-twitter/src/test/java/org/apache/streams/twitter/processor/FetchAndReplaceTwitterProcessorTest.java]: Small nit: You've got the params in the wrong order.  It's expected then actual.

**Comment by robdouglas** [comment]
Reversed assert parameters

**Comment by sieverssj** [comment]
:+1:

**Pull request merged**